aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,101 @@
+import { Example } from '../../layouts';
+import { 
+  PageBackGroundExample,
+  PageExample, 
+  PageFullExample,
+  PageNarrowExample,
+  PageWideExample 
+} from '../../examples';
+
+Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE applications. Two components, Page and PageContent, work together to ensure HPE theme-defined margins, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, turn-key manner.
+
+<Example 
+  showResponsiveControls={['fullScreen']}
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+>
+  <PageExample />
+</Example>
+
+## Guidance
+
+Page is the outer layout component. Page’s `kind` determines which theme-defined padding, min/max-width, and alignment should be applied to its child PageContent components.
+
+PageContent should be a child of Page. This allows PageContent to adhere to the min/max-width, padding, and alignment values set by its parent Page `kind`.
+
+In most cases, all of the content for a page can be wrapped in a single PageContent component. However, if varying background colors are needed for different page sections or a full-width background color is needed
+(link to variant below), then individual page sections can each be wrapped in their own PageContent.
```

Comment:
I'm guessing there should be a link here

Reply:
oh yes